crates/test-programs/Cargo.toml

```diff
@@ -20,3 +20,4 @@ futures = { workspace = true, default-features = false, features = ['alloc'] }
 url = { workspace = true }
 sha2 = "0.10.2"
 base64 = "0.21.0"
+image2tensor = { git = "https://github.com/bytecodealliance/wasi-nn.git", rev = "dec0dd4" }
```

Comment:
It looks like `cargo vet` doesn't like this git link. We'll need to release a new version for image2tensor and update the version number here.

Reply:
This dependency has been removed. Test image was converted to tensor data offline.